src/openrct2/cheats.c

```diff
@@ -428,6 +430,64 @@ static void cheat_set_staff_speed(uint8 value)
 	}
 }
 
+static void cheat_own_all_land()
+{
+	sint32 min = 32, max = gMapSizeUnits - 32;
+	for (sint32 y = min; y <= max; y += 32) {
+		for (sint32 x = min; x <= max; x += 32) {
+			rct_map_element* surfaceElement = map_get_surface_element_at(x / 32, y / 32);
+
+			// Ignore already owned tiles
+			if (surfaceElement->properties.surface.ownership & OWNERSHIP_OWNED ||
+				surfaceElement->properties.surface.ownership & OWNERSHIP_CONSTRUCTION_RIGHTS_OWNED)
+				continue;
+
+			sint32 base_z = surfaceElement->base_height;
+			rct_map_element* mapElement = map_get_first_element_at(x / 32, y / 32);
+			sint32 destOwnership = OWNERSHIP_OWNED;
+			
+			do {
+				sint32 type = map_element_get_type(mapElement);
+				// do not own park entrance tiles
+				if (type == MAP_ELEMENT_TYPE_ENTRANCE) {
+					destOwnership = 0;
+				}
+				// set all tiles containing paths to construction rights
+				else if (type == MAP_ELEMENT_TYPE_PATH) {
+					destOwnership = OWNERSHIP_CONSTRUCTION_RIGHTS_OWNED;
+					// Do not own paths too high/below surface
+					if (mapElement->base_height - 4 > base_z ||
+						mapElement->base_height < base_z) {
+						destOwnership = 0;
```

Comment:
the reason why it does that is so the while will be triggered false. "while (destOwnership != 0)"